### PR TITLE
"Add blog post about ascii art in elixir."

### PR DESCRIPTION
**Pull request by curiosdevcookie**
- Add gitignore file.
- Add blog post about ascii art in elixir.

### CODE REVIEW

File: livebooks/elixir_ASCII_heart.livemd [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+The following code achieves that by using a combination of mathematical equations and string manipulation techniques.
+
+You'll find a step-by-step breakdown after the code. Let's go!
+
+```elixir
+defmodule Love do
+  @love "Elixir "
+
+  def character_at(@love, x, y) do
```
Comment (resterle) [inline livebooks/elixir_ASCII_heart.livemd]: Why have @love as parameter here?

File: livebooks/elixir_ASCII_heart.livemd [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+end
+
+IO.puts(Love.heart())
+```
+
+## Let's break it down
+
+The code is split into 5 parts:
+
+1. The definition of the module `WeLove` contains the global variable `love`, and the functions `character_at` and `heart`.
```
Comment (resterle) [inline livebooks/elixir_ASCII_heart.livemd]: Module attribute -> https://elixir-lang.org/getting-started/module-attributes.html

File: livebooks/elixir_ASCII_heart.livemd (hunk 1 above)
Comment (curiosdevcookie) [reply livebooks/elixir_ASCII_heart.livemd]: I'll remove it :-)

File: livebooks/elixir_ASCII_heart.livemd (hunk 2 above)
Comment (curiosdevcookie) [reply livebooks/elixir_ASCII_heart.livemd]: Do you mean, I should call @love a module attribute?

File: livebooks/elixir_ASCII_heart.livemd [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+```elixir
+defmodule Love do
+  @love "Elixir "
+
+  def character_at(@love, x, y) do
+    index = rem(x - y, String.length(@love))
+    char = String.slice(@love, index..index)
+
+    if (:math.pow(x * 0.05, 2) + :math.pow(y * 0.1, 2) - 1) ** 3 -
```
Comment (resterle) [inline livebooks/elixir_ASCII_heart.livemd]: For better readability i would put that in its own function then you can write if(something?) do.....

Comment (resterle) [reply livebooks/elixir_ASCII_heart.livemd]: or even case something? do true -> ... _other -> ....

**Review by resterle — APPROVED** [review]

**Pull request merged**